automod/triggers.go

```diff
@@ -1143,6 +1190,12 @@ func (nwl *NicknameWordlistTrigger) UserSettings() []*SettingDef {
 			Key:  "ListID",
 			Kind: SettingTypeList,
 		},
+		{
+			Name:    "Also match Zalgo and confusable characters",
```

Comment:
Make it simple for a average user? What does Zalgo and confusable characters here mean?

Also, I see it being repeated a lot in this file, maybe create a single variable and use it everywhere?

Reply:
That would definitely work to just have it as a variable to easily change